packages/mds-logger/index.ts

```diff
@@ -29,8 +31,9 @@ const log = (level: LogLevel, ...args: unknown[]): string[] => {
   if (redacted.length) {
     const timestamp = Date.now()
     const ISOTimestamp = new Date(timestamp).toISOString()
+    const requestId = httpContext.get('x-request-id')
```

Comment:
What happens when `mds-logger` is used outside of an http request context?

Reply:
Just logs `undefined`. I already marked that as something for me to fix, this is just the result of super hacky prototyping (hence being in draft)

Reply:
As of recent cleanup commits, now doesn't log it at all if non-existant